test/Azure.Functions.Cli.Tests/E2E/InitTests.cs

```diff
@@ -208,6 +208,36 @@ public Task init_dotnet_app()
             }, _output);
         }
 
+        [Fact]
+        public Task init_dotnet_app_net9_isolated()
+        {
+            return CliTester.Run(new RunConfiguration
+            {
+                Commands = new[] { "init dotnet-funcs --worker-runtime dotnet-isolated --target-framework net9.0" },
+                CheckFiles = new[]
+                {
+                    new FileResult
+                    {
+                        Name = Path.Combine("dotnet-funcs", "local.settings.json"),
+                        ContentContains = new[]
+                        {
+                            "FUNCTIONS_WORKER_RUNTIME",
+                            "dotnet-isolated",
+                        }
+                    },
+                    new FileResult
+                    {
+                        Name = Path.Combine("dotnet-funcs", "dotnet-funcs.csproj"),
+                        ContentContains = new[]
+                        {
+                            "Microsoft.NET.Sdk",
```

Comment:
Would the csproj have something like "Microsoft.NET.Sdk" ? I am not following what this part is verifying.

Reply:
Yeah it does in the first part of the csproj:
```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>net9.0</TargetFramework>
    <AzureFunctionsVersion>v4</AzureFunctionsVersion>
    <OutputType>Exe</OutputType>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
  </PropertyGroup>
```

Reply:
Got it. may be we should also check for `TargetFramework` value.

Reply:
Added a check for `TargetFramework`